ddtrace/internal/remoteconfig/_connectors.py

```diff
@@ -71,15 +73,18 @@ def read(self) -> SharedDataType:
         config = json.loads(config_raw) if config_raw else None
         if config is not None:
             shared_data_counter = config["shared_data_counter"]
-            if shared_data_counter > self.shared_data_counter:
-                self.shared_data_counter += 1
+            if os.getpid() != self.pid:
```

Comment:
I am not sure I understand this change. I understand what the code is actually doing, but the impact is not clear to me.

we don't seem to be incrementing the counter anymore?

but also down below it is now a monotonic timestamp?


I might not be able to review this specific change to RC since I am not the most familiar with the code/protocol/etc.